core/src/main/java/org/mule/endpoint/AbstractEndpoint.java

```diff
@@ -508,6 +508,12 @@ public void dispose()
         // this.messageProcessors.clear();
         // Don't clear this, since it changes the hash code, which can foul up shutdown processing
         // when objects have been keyed by endpoint, e.g. dispatchers
+
```

Comment:
Put the new lines before the comment, so the comment is right next to the lines where it applies